### ISSUE TITLE
Failed to fetch monocular file

### ISSUE THREAD

**Issue by hwshin16** [helm/monocular#479]
Hello, I saw this error message during executing commands.
My repo url is "monocular	https://helm.github.io/monocular".

$ helm install --name monocular monocular/monocular --debug 

[debug] Created tunnel using local port: '49379'

[debug] SERVER: "127.0.0.1:49379"

[debug] Original chart version: ""
Error: Failed to fetch https://kubernetes-helm.github.io/monocular/monocular-0.6.3.tgz : 404 Not Found

**Comment by jimleitch01** [comment]
Hi hwshin16,

I worked around this (on MacOS) by:

```
rm -r ~/.helm
helm init
helm repo add monocular https://helm.github.io/monocular
helm repo update
vi /$YOURHOMEDIRHERE/.helm/repository/cache/monocular-index.yaml
```
Search and replace `kubernetes-helm` with `helm`

```
 helm install monocular/monocular
``` 

Not beautiful but gets the job done

**Comment by hwshin16** [comment]
Thank you for reply, jumleitch01. I did it, but now I see "CrashLoopBackOff" status of monocular-api and  "Pending" status of monocular-mongodb.
What's happening?

**Comment by jroucheton** [comment]
Same issue.
index.yaml for Helm repository seems to be corrupted.
As you can see on the following extract, URL is not up-to-date.

It should be **https://helm.github.io/monocular/monocular-0.6.3.tgz** instead of **https://kubernetes-helm.github.io/monocular/monocular-0.6.3.tgz**


apiVersion: v1
entries:
  monocular:
  - apiVersion: v1
    appVersion: v0.7.3
    created: 2018-07-13T18:36:49.210303598Z
    description: Monocular is a search and discovery front end for Helm Charts Repositories.
    digest: 52b88ec22754e88a3ed0f4c67921bab840dae9727224779887abaf02023fb2cf
    home: https://github.com/kubernetes-helm/monocular
    maintainers:
    - email: adnan@bitnami.com
      name: prydonius
    name: monocular
    sources:
    - https://github.com/kubernetes-helm/monocular
    urls:
    - **https://kubernetes-helm.github.io/monocular/monocular-0.6.3.tgz**
    version: 0.6.3`

**Comment by prydonius** [comment]
fixed in #480 thanks all!
